### DISCUSSION TITLE
Need help deleting unverified account

### DISCUSSION THREAD

**Discussion by keithradford** [vercel/community#3887] [Account Issue]
### Question

In attempt to deploy to an environment at my new job, I tried to create a new account on vercel with my work email. Since I couldn't verify it, as my phone number is tied to my personal account, I can't change my email on my personal account to my work email.

If possible I would like the unverified account deleted so I can be unblocked in changing my email on my existing account. Will provide more info if needed, thanks.

**Comment by amyegan** [comment]
Hey @keithradford. Sorry you had trouble signing up for a new account. We can't safely discuss account information in public because of security concerns. Please email [registration@vercel.com](mailto:registration@vercel.com) and include the details of the account you need to delete. That will open a new ticket with the team that handles account issues.

**Reply by keithradford** [reply]
Hi there, I sent an email a few days back. Wondering if it can be escalated at all as it is blocking my work. Thanks.
